README.md

```diff
@@ -63,7 +63,7 @@ It contains hundreds of controls covering CIS, NIST 800, NIST CSF, CISA, RBI, Fe
 
 | Provider | Checks | Services | [Compliance Frameworks](https://docs.prowler.com/projects/prowler-open-source/en/latest/tutorials/compliance/) | [Categories](https://docs.prowler.com/projects/prowler-open-source/en/latest/tutorials/misc/#categories) |
 |---|---|---|---|---|
-| AWS | 383 | 67 -> `prowler aws --list-services` | 28 -> `prowler aws --list-compliance` | 7 -> `prowler aws --list-categories` |
```

Comment:
Why this?

Reply:
I think that is the actual number of AWS checks in master, I used this command to check it: `find ./prowler/providers/aws/services -mindepth 2 -maxdepth 2 -type d | grep -Ev "__pycache__|lib" | wc -l`

Reply:
I would change it only when `prowler aws -l` also gives that number.

Reply:
Could you take a look to see what is happening?

Reply:
Yeah sure, I will study what could be happening.

Reply:
I found the following checks with the same problem:
- awslambda_function_not_directly_publicly_accessible_via_elbv2
- cloudtrail_threat_detection_enumeration
- cloudtrail_threat_detection_privilege_escalation
- cognito_identity_pool_guest_access_enabled
- cognito_user_pool_advanced_security_adaptative_authentication_block_sign_in
- cognito_user_pool_advanced_security_block_sign_in_compromised_credentials
- cognito_user_pool_prevent_reveal_user_existence
- cognito_user_pool_self_registration_enabled
- cognito_user_pool_strong_password_policy
- cognito_user_pool_token_revocation_enabled
- cognito_user_pool_web_acl_associated
- dms_instance_multi_az
- documentdb_cluster_backup_enabled
- documentdb_cluster_deletion_protection
- documentdb_cluster_deletion_protection_enabled
- documentdb_instance_storage_encrypted
- ec2_instance_port_elasticsearch_kibana_exposed_to_internet
- ec2_instance_port_memcached_exposed_to_internet
- ec2_instance_port_mysql_exposed_to_internet
- neptune_cluster_backup_enabled
- neptune_cluster_deletion_protection
- neptune_cluster_multi_az
- neptune_cluster_storage_encrypted